task.sql

```diff
@@ -1,11 +1,19 @@
 -- Use our database
 USE ShopDB; 
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, CURRENT_DATE);
```

Comment:
The `INSERT INTO Orders` statement is executed before the transaction is started. If this is intended to be part of the transaction, it should be moved after the `START TRANSACTION` statement.